### PR TITLE
Send reaction method placeholder

### PR DESCRIPTION
**Pull request by paleksandrs**
- Placeholder for send reaction method (https://github.com/matrix-org/matrix-rust-sdk/pull/1257)
- Removed unnecessary emoji skin parsing (new emoji JSON contains native representation of the emoji)

### CODE REVIEW

File: ElementX/Sources/Screens/RoomScreen/RoomScreenCoordinator.swift [hunk 1]
```diff
@@ -113,20 +113,23 @@ final class RoomScreenCoordinator: CoordinatorProtocol {
     }
     
     private func displayEmojiPickerScreen(for itemId: String) {
-        guard let emojiProvider = parameters?.emojiProvide else {
+        guard let emojiProvider = parameters?.emojiProvide,
```
Comment (stefanceriu) [inline ElementX/Sources/Screens/RoomScreen/RoomScreenCoordinator.swift:116]:
```suggestion
        guard let emojiProvider = parameters?.emojiProvider,
```

File: ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineItemReactionsMenuView.swift [hunk 2]
```diff
@@ -19,13 +19,17 @@ import SwiftUI
...
     
+    var onEmojiSelected: ((String) -> Void)?
     var onDisplayEmojiPicker: (() -> Void)?
     
     var body: some View {
         HStack {
             HStack(spacing: 10) {
                 ForEach(emojis, id: \.self) { emoji in
                     Text(emoji)
+                        .onTapGesture {
```
Comment (stefanceriu) [inline ElementX/Sources/Screens/RoomScreen/View/Supplementary/TimelineItemReactionsMenuView.swift:30]: Shouldn't these be wrapped in Buttons now?

**Review by stefanceriu — APPROVED** [review]
Looks good to me, a few suggestions inline 👍
Seems there's also a linter error in the `RoomTimelineController`, mind fixing that as well please?

**Comment by paleksandrs** [comment]
@stefanceriu thanks for the comments. All fixed 👍

**Comment by stefanceriu** [comment]
Nice, thanks!

**Pull request merged**